src/tests/Sessions.test.ts

```diff
@@ -1,25 +1,26 @@
 import { AvoBatcher } from "../AvoBatcher";
 import { AvoInspector } from "../AvoInspector";
-import { AvoInspectorEnv } from "../AvoInspectorEnv";
 import { AvoNetworkCallsHandler } from "../AvoNetworkCallsHandler";
 import { AvoSessionTracker } from "../AvoSessionTracker";
 import { AvoStorage } from "../AvoStorage";
 
+import { defaultOptions, sessionTimeMs } from "../__tests__/constants";
+const inspectorVersion = process.env.npm_package_version || "";
+
 jest.mock("../AvoBatcher");
 jest.mock("../AvoNetworkCallsHandler");
 
-const defaultOptions = {
-  apiKey: "api-key-xxx",
-  env: AvoInspectorEnv.Prod,
-  version: "0",
-};
-
-// TODO: Reuse in class implementation and test
-const sessionTimeMs = 5 * 60 * 1000;
```

Comment:
You ended up having separate constants in tests and in the app, right? I think it makes sense to have them separate.

Reply:
Yes, that's right, created `constants.ts` file that stores all the constants that are used only in the tests and currently these values are not shared between tests and classes. So will leave it like that